lib/ui/platform_dispatcher.dart

```diff
@@ -857,6 +867,11 @@ class PlatformDispatcher {
 
     final double textScaleFactor = (data['textScaleFactor'] as num).toDouble();
     final bool alwaysUse24HourFormat = data['alwaysUse24HourFormat'] as bool;
+    // This field is optional.
```

Comment:
If it's null, does that mean that the engine hasn't been able to determine its value yet?

Reply:
On Android the field should always be present.

Reply:
(because I don't want to add this for other platforms)